enterprise/app/models/captain/document.rb

```diff
@@ -26,11 +28,14 @@ class LimitExceededError < StandardError; end
   belongs_to :assistant, class_name: 'Captain::Assistant'
   has_many :responses, class_name: 'Captain::AssistantResponse', dependent: :destroy, as: :documentable
   belongs_to :account
+  has_one_attached :file
 
   validates :external_link, presence: true
   validates :external_link, uniqueness: { scope: :assistant_id }
-  validates :content, length: { maximum: 200_000 }
+  validates :content, length: { maximum: 400_000 }
+  validates :source_type, inclusion: { in: %w[url pdf_upload] }, allow_blank: true
```

Comment:
We can use an enum here

```rb
enum :source_type, %w[ pdf url ].index_by(&:itself), default: 'url', suffix: true, validate: true
```

That will ensure the string type remains and the `suffix: true` will create methods like `pdf_source_type?` and `url_source_type?` 
In the future we can add notion and google_docs as well